ai_diffusion/language/es.json

```diff
@@ -0,0 +1,376 @@
+{
+  "id": "es",
+  "name": "español",

```

Comment:
Please change first letter to upper case

Reply:
I'm sorry about that. that's ok now.